vector/src/main/java/im/vector/app/features/permalink/PermalinkHandler.kt

```diff
@@ -122,6 +123,13 @@ class PermalinkHandler @Inject constructor(private val activeSessionHolder: Acti
         }
     }
 
+    private fun isPermalinkSupported(context: Context, url: String): Boolean {
+        return url.startsWith(PermalinkService.MATRIX_TO_URL_BASE)
+                || context.resources.getStringArray(R.array.permalink_supported_hosts).any {
```

Comment:
idem "Don't you think we should consider the case of "clientPermalinkBaseUrl" here too?"

Reply:
fixed by https://github.com/vector-im/element-android/pull/4027/files#diff-47e155bb67cd55047db56ba0b5d2853eb4dd79f5e1bb07b566b6b7ba31efd7f0R38